src/lib.rs

```diff
@@ -469,8 +469,8 @@ impl<K: Eq + Hash, V, S: BuildHasher> TtlCache<K, V, S> {
         }
     }
 
-    fn remove_oldest(&mut self) -> Option<(K, V)> {
```

Comment:
I'd prefer to keep this method signature the same.  It makes someone might want to see the value that as its removed.

Reply:
The method is not public so only users of the method are contributors to the crate itself. When the need for removed value arrives this return value can of course be introduced again.

Also maybe the compiler optimizes the return value away because it's not used so this change might only be cosmetic.

I can remove the commit and update PR if you feel strongly about this :)

Reply:
I removed and now committed this change back. I'm good with this PR now.